### ISSUE TITLE
18+ Cookie

### ISSUE THREAD

**Issue by JPH71** [acvigue/pgma#17]
Hi aiden

You mentioned in one of the threads that one could emulate the download of a cookie to fool the scrapper that one has clicked on the button that then allows you to search the website...

If you have any time - could you let me know how to do this so that I can go ahead and work on maleflixxtv

Kind Regards

Jason

**Comment by acvigue** [comment]
Working on it right now.

**Comment by acvigue** [comment]
Try adding this line to the `start()` function in init.py

`HTTP.Headers['Cookie'] = 'Cookie: maleflixx.tvtaste=M'`

**Comment by JPH71** [comment]
Thanks Aiden for this

Could you let me know how you determine this so that I can do the same for
GayVODCLub?

Kind Regards

**Comment by acvigue** [comment]
looked at what cookies changed when the “enter” button was clicked and set the http header

**Comment by JPH71** [comment]
Maybe I should have asked - where do i look to find the information
being a bit of an idiot here...

Jason x

**Comment by acvigue** [comment]
of course, you are fine. it’s in the storage tab of the developer console (at least in Firefox) but you open the Storage display and cljck the button and see what changes

**Comment by JPH71** [comment]
Thanks for this – I just realised fairly recently how wonderful the console is in Chrome and fireofx once you inspect an element...

Before that it took ages to test any xpath expressions and nearly 2 weeks to right a scrapper....

Good night

Jason x

Sent from Mail for Windows 10

**Comment by JPH71** [comment]
Hi Aiden

I followed your instruction, but the setting of the headers does not seem to work.

Could you please look at my code in case I am doing something extraordinary stupid?

Cheers Man

[GayVODClub.bundle.zip](https://github.com/acvigue/pgma/files/4119871/GayVODClub.bundle.zip)

**Comment by acvigue** [comment]
Theoretically, your code should work. I looked at it, but I can look up alternate solutions to get around setting the cookie.
